### PR TITLE
Fixed links for storage drivers

### PR DESCRIPTION
**Pull request by bobrik**
Side note: after reading docs it seems like v2 is still far in the future and incomplete. It make sense to proof-read docs by someone who wasn't working on the spec directly to fix that.

### CODE REVIEW

**Comment by stevvooe** [comment]
LGTM once the commit is signed!

@bobrik That is a valid criticism. Would you like to volunteer to help out with proofreading?

cc @moxiegirl

**Comment by bobrik** [comment]
You should probably invite @gordontheturtle to remind about signing. Should be good now.

**Comment by stevvooe** [comment]
@bobrik Gordon the turtle plans to crawl all over this project. Thanks for the update.

@moxiegirl Please confirm these changes won't break docs.

**Comment by moxiegirl** [comment]
@stevvooe Sorry, this one breaks the docs.  Currently, we aren't building these files into the docs.docker.com --- I need to come up with a way to make it work in both places.

@bobrik Thanks for the contribution.  I'll have a new file for you to add into your PR shortly.

**Comment by moxiegirl** [comment]
@bobrik To ensure the build doesn't break,  you need to update the `mkdocs.yml` file.   Rebase to pull in the latest file from master, then make it look like this:

```

- ['registry/index.md', 'Reference', 'Docker Registry 2.0']
- ['registry/deploying.md', 'Reference', '&nbsp;&nbsp;&nbsp;&nbsp;&blacksquare;&nbsp; Deploy a registry' ]
- ['registry/configuration.md', 'Reference', '&nbsp;&nbsp;&nbsp;&nbsp;&blacksquare;&nbsp; Configure a registry' ]
- ['registry/storagedrivers.md', 'Reference', '&nbsp;&nbsp;&nbsp;&nbsp;&blacksquare;&nbsp; Storage driver model' ]
- ['registry/notifications.md', 'Reference', '&nbsp;&nbsp;&nbsp;&nbsp;&blacksquare;&nbsp; Work with notifications' ]
- ['registry/spec/api.md', 'Reference', '&nbsp;&nbsp;&nbsp;&nbsp;&blacksquare;&nbsp; Registry Service API v2' ]
- ['registry/spec/json.md', 'Reference', '&nbsp;&nbsp;&nbsp;&nbsp;&blacksquare;&nbsp; JSON format' ]
- ['registry/spec/auth/token.md', 'Reference', '&nbsp;&nbsp;&nbsp;&nbsp;&blacksquare;&nbsp; Authenticate via central service' ]

- ['registry/storage-drivers/azure.md', '**HIDDEN**' ]
- ['registry/storage-drivers/filesystem.md', '**HIDDEN**' ]
- ['registry/storage-drivers/inmemory.md', '**HIDDEN**' ]
- ['registry/storage-drivers/s3.md', 'Reference', '**HIDDEN**' ]

```

The blank line at the end is important.  You can run `make docs` from the root of your repo to test.

**Comment by moxiegirl** [comment]
@bobrik If you don't want to finish this out, I'm happy to carry it for you. If we don't hear from you in a few days, I'll do that.  Cheers.

**Comment by bobrik** [comment]
@moxiegirl np, take a look now. I'm not sure if you meant C-like empty line at the end of file or an extra one. I used the former, links seemed to be working.

File: docs/mkdocs.yml [hunk 1]
```diff
@@ -7,3 +7,7 @@
 - ['registry/spec/json.md', 'Reference', '&nbsp;&nbsp;&nbsp;&nbsp;&blacksquare;&nbsp; JSON format' ]
 - ['registry/spec/auth/token.md', 'Reference', '&nbsp;&nbsp;&nbsp;&nbsp;&blacksquare;&nbsp; Authenticate via central service' ]
 
+- ['registry/storage-drivers/azure.md', '**HIDDEN**' ]
+- ['registry/storage-drivers/filesystem.md', '**HIDDEN**' ]
+- ['registry/storage-drivers/inmemory.md', '**HIDDEN**' ]
+- ['registry/storage-drivers/s3.md', 'Reference', '**HIDDEN**' ]
```
Comment (moxiegirl) [inline docs/mkdocs.yml]: Yikes,  passed you bad data.  Can you change this line to :

+- ['registry/storage-drivers/s3.md', '**HIDDEN**' ]

**Comment by moxiegirl** [comment]
One minor remaining tweak which was totally my fault. Stellar work @bobrik thank you for the PR!

LGTM

**Comment by bobrik** [comment]
Check again :)

**Comment by moxiegirl** [comment]
Wow.

**Comment by bobrik** [comment]
Build failed because of coveralls

**Comment by dmp42** [comment]
Kicked the build + planning on not making the build fail because of coveralls.

Merged and thanks @bobrik !

**Comment by bobrik** [comment]
:whale2: :tada:

**Pull request merged**